api/mysagw/identity/management/commands/upload_labels_template.py

```diff
@@ -0,0 +1,39 @@
+from pathlib import Path
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from requests import HTTPError
+
+from mysagw.identity.dms_client import DMSClient
+
+
+class Command(BaseCommand):
+    help = "Upload template for printing address labels"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.client = DMSClient()
+
+    def _upload_template(self, update=False):
+        try:
+            self.client.upload_template(
+                settings.DOCUMENT_MERGE_SERVICE_LABELS_TEMPLATE_SLUG,
+                "Identity address labels",
+                (
+                    Path(__file__).parent.parent.parent.absolute()
```

Comment:
Could we convert this into a generic upload tool? Such that other templates can be uploaded as well? I assume this will be used to prime the DMS during setup / update. If we did that, adding another file would be just another command call

Reply:
I've made it a bit more generic now. Generally this is only used for the templates that are managed by us (right now only one). There will be a possibility for staff users to manage their templates from the frontend.